.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 23.9.1  # Use the latest release of black
+    hooks:
+      - id: black
+        language_version: python3  # only choose python 3
```

Comment:
Do we need to further specify the python version, or is python3 enough?

Reply:
Yeah, so good idea @davidrudlstorfer, I think we can add (or forbid) some more functionalities. However, I would try not overcomplicate things too much, but your repo provides a good start. 
@isteinbrecher any additional features you like ?

Reply:
I am fine with having `black` in the default settings (especially the increased number of characters per line).

@davidrudlstorfer is the `python3.12` actually specify a python version that is used to run the pre-commit hook or is this sort of an option for black to know how to format the code?

Thanks for the link @davidrudlstorfer! I don't know most of the tools used there, but I am sure some could be very useful for MeshPy. I will open up an issue to gather ideas on possible advances in that direction.

Reply:
@isteinbrecher the `language_version: python 3.xx` should coincide with the installed/used python version. 

Therefore, it would be beneficial to initially solve #76 and once and for all define a specific Python version for MeshPy before adapting any pre-commit hooks

Reply:
I opened a pull request to resolve #76. Then we can put python3.12 here.

Reply:
#76 is solved, so I think we can proceed here.